README.md

```diff
@@ -694,6 +696,11 @@ If there are actions for a webhook, events are emitted for both, the webhook nam
 
 <!-- /autogenerated via scripts/generate-types.ts -->
 
+### emitterEventNames
+
+The read only tuple containing all the possible webhook event names listed above.
+This might be useful in GUI and input validation.
```

Comment:
It's not only the event names, but the combination of event names and actions. How about this?

````suggestion
A read only tuple containing all the possible combinations of the webhook events + actions listed above. This might be useful in GUI and input validation.

```js
import { emitterEventNames } from "@octokit/webhooks";
emitterEventNames // ["check_run", "check_run.completed", ...]
```
````